### ISSUE TITLE
camera path is not taken into account when rendering videos

### ISSUE THREAD

**Issue by Justus03** [nerfstudio-project/nerfstudio#3401]
**Describe the bug**
Hi everyone! I'm having trouble rendering videos of my 3dgs scene inside nerfstudio's viewer. Whenever I configure the camera path for the animation and generate the command it gives me a video where the camera stays on the same position and just keep rotating...

command:
ns-render camera-path --load-config outputs/output/splatfacto/2024-08-30_140322/config.yml --camera-path-filename /root/img\ set/output/camera_paths/2024-08-30-14-03-35_2.json --output-path renders/output/2024-08-30-14-03-35_3.mp4

camera path:
![Screenshot_20240830_160358](https://github.com/user-attachments/assets/9410e94f-25dc-4543-892c-fc11b9b72bf4)

Output:
https://github.com/user-attachments/assets/bc04273a-18da-40c2-b848-fdb5cb7d15d9

**Comment by nirmalsnair** [comment]
Hi @Justus03, did you manage to solve this issue?

**Comment by Justus03** [comment]
> Hi [@Justus03](https://github.com/Justus03), did you manage to solve this issue?

Not really... I just switched to a Pixi installation instead and everything worked fine
